addon/components/table-columns.js

```diff
@@ -58,12 +60,17 @@ export default Ember.Component.extend({
   }),
 
   /**
-    Gets table's rowHeight.
+    Sets an inline style for height on all table rows from table.rowHeight.
     @public
-    @default DEFAULT_ROW_HEIGHT
   */
   rowHeight: computed('table.rowHeight', function() {
-    return getWithDefault(this, 'table.rowHeight', DEFAULT_ROW_HEIGHT);
+    let rowHeight = this.get('table.rowHeight');
+
+    if (rowHeight) {
+      return new Ember.Handlebars.SafeString(`height: ${rowHeight}`);
+    }
+
+    return new Ember.Handlebars.SafeString('');
```

Comment:
`vertical-collection` needs a number rather than a string, so the original line would be correct here.